zerver/tests/test_upload.py

```diff
@@ -1190,3 +1191,24 @@ def test_file_name(self) -> None:
         self.assertEqual(sanitize_name(u'snowman☃.txt'), u'snowman.txt')
         self.assertEqual(sanitize_name(u'테스트.txt'), u'테스트.txt')
         self.assertEqual(sanitize_name(u'~/."\`\?*"u0`000ssh/test.t**{}ar.gz'), u'.u0000sshtest.tar.gz')
+
+
+class UploadSpaceTests(UploadSerializeMixin, ZulipTestCase):
+    def setUp(self) -> None:
+        self.realm = get_realm("zulip")
+        self.user_profile = self.example_user('hamlet')
+
+    def test_currently_used_upload_space_for_realm_with_no_uploads(self) -> None:
+        self.assertEqual(0, currently_used_upload_space(self.realm))
+
+    def test_currently_used_upload_space_for_realm_with_one_upload(self) -> None:
+        data = b'zulip!'
+        upload_message_file(u'dummy.txt', len(data), u'text/plain', data, self.user_profile)
+        self.assertEqual(len(data), currently_used_upload_space(self.realm))
+
+    def test_currently_used_upload_space_for_realm_with_many_uploads(self) -> None:
+        data1 = b'zulip!'
+        data2 = b'more-data!'
+        upload_message_file(u'dummy1.txt', len(data1), u'text/plain', data1, self.user_profile)
+        upload_message_file(u'dummy2.txt', len(data2), u'text/plain', data2, self.user_profile)
+        self.assertEqual(len(data1) + len(data2), currently_used_upload_space(self.realm))
```

Comment:
It feels a bit unnecessary to put each of these in their own tests; a running test like this should be fine:

```
diff --git a/zerver/tests/test_upload.py b/zerver/tests/test_upload.py
index 4296630..33a07a5 100644
--- a/zerver/tests/test_upload.py
+++ b/zerver/tests/test_upload.py
@@ -1198,17 +1198,13 @@ class UploadSpaceTests(UploadSerializeMixin, ZulipTestCase):
         self.realm = get_realm("zulip")
         self.user_profile = self.example_user('hamlet')
 
-    def test_currently_used_upload_space_for_realm_with_no_uploads(self) -> None:
+    def test_currently_used_upload_space(self) -> None:
         self.assertEqual(0, currently_used_upload_space(self.realm))
 
-    def test_currently_used_upload_space_for_realm_with_one_upload(self) -> None:
         data = b'zulip!'
         upload_message_file(u'dummy.txt', len(data), u'text/plain', data, self.user_profile)
         self.assertEqual(len(data), currently_used_upload_space(self.realm))
 
-    def test_currently_used_upload_space_for_realm_with_many_uploads(self) -> None:
-        data1 = b'zulip!'
         data2 = b'more-data!'
-        upload_message_file(u'dummy1.txt', len(data1), u'text/plain', data1, self.user_profile)
         upload_message_file(u'dummy2.txt', len(data2), u'text/plain', data2, self.user_profile)
-        self.assertEqual(len(data1) + len(data2), currently_used_upload_space(self.realm))
+        self.assertEqual(len(data) + len(data2), currently_used_upload_space(self.realm))
```